src/bin/p-s.test.js

```diff
@@ -0,0 +1,53 @@
+import path from 'path'
+import test from 'ava'
+import tester from 'cli-tester';
+import fs from 'fs';
+import os from 'os';
+import {createHash} from 'crypto';
+
+
+const babel = require.resolve('babel-cli/bin/babel-node');
+const cli = require.resolve('./p-s');
+
+
+const hash = string => createHash('md5').update(string).digest('hex');
+const tmpConfig = string => path.join(os.tmpdir(), `tmp-config-${hash(string)}.js`);
+const writeConfig = (file, string) => new Promise(resolve => fs.writeFile(file, string, resolve));
+const rmConfig = (file) => new Promise(resolve => fs.unlink(file, resolve));
+
+
+test('p-s: run without args', t =>
+  tester(babel, require.resolve('./p-s'))
```

Comment:
Kind of weird way to babelify scripts... I thought cli-tester could be updated, but not sure how exactly update it to avoid any extra complexity.

Same for config files. This looks like a nice thing to abstract. Maybe I will add these as just helpers that one can use.